e2e-tests/cypress/integration/appeal-submission-tasklist.feature

```diff
@@ -0,0 +1,28 @@
+@wip
+Feature: Your Details status
+
+    Scenario Outline: The status of the Your Details section is accurate
+        Given the user has provided their name: <name> and email: <email>
+        And the user has stated that they "are" the original appellant
+        When the user checks the status of their appeal
+        Then the user should see that the Your Details section is <status>
+
+        Examples:
+            | name        | email            | Status        |
+            | ""          | ""               | "Not Started" |
+            | ""          | "good@email.com" | "Not Started" |
+            | "Good Name" | ""               | "Not Started" |
+            | "Good Name" | "good@email.com" | "Completed"   |
+
+
+      Scenario Outline: The status of the Your Details section is accurate
```

Comment:
I think we could use a different wording in the title to show it is different from the scenario above i.e. in this case "... when the user is not the original applicant"

Reply:
Will offer tidy up in glue code

Reply:
This will be addressed while implementing the glue code